src/main/kotlin/com/ampnet/reportserviceeth/blockchain/BlockchainServiceImpl.kt

```diff
@@ -28,13 +31,13 @@ class BlockchainServiceImpl(private val applicationProperties: ApplicationProper
 
     override fun getIssuerOwner(issuer: String): String {
         logger.debug { "Get owner of issuer: $issuer" }
-        TODO("Not implemented")
+        val contract = IIssuer.load(issuer, web3j, readonlyTransactionManager, DefaultGasProvider())
+        return contract.state.send().owner
```

Comment:
`send()` function can throw an exception, handle it. Do the same for other blockchain calls.